sdk-python/littlehorse/workflow.py

```diff
@@ -493,6 +511,55 @@ def __str__(self) -> str:
         return to_json(self.compile())
 
 
+class SpawnedThread:
+    def __init__(self, name: str, number: WfRunVariable) -> None:
+        self.name = name
+        self.number = number
+
+
+class SpawnedThreads:
+    def __init__(
+        self,
+        iterable: Optional[WfRunVariable],
+        fixed_threads: Optional[list[SpawnedThread]] = None,
+    ) -> None:
+        self._iterable = iterable
+        self._fixed_threads = fixed_threads
+
+    @classmethod
+    def from_list(cls, *spawned_threads: SpawnedThread) -> "SpawnedThreads":
+        return SpawnedThreads(iterable=None, fixed_threads=list(spawned_threads))
+
+    def build_node(self) -> WaitForThreadsNode:
```

Comment:
WaitForThreadsNode is a proto? can you change this method name to compile()?

Reply:
Good suggestion. thanks.